CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### New Features
 
+* [#856](https://github.com/toptal/chewy/pull/856): Fix return value of subscribed_task_stats used in rake tasks. ([@fabiormoura][])
+
 ### Changes
 
 ### Bugs Fixed
```

Comment:
I would say it's a bug fix, rather than new feature :) but I'll fix it in the release.

Reply:
It was because I haven't noticed there was a specific section in this file for bug fixes but thanks for approving the PR with this small mistake.

Reply:
this has been resolved.